internal/resources/metal/project_ssh_key/resource.go

```diff
@@ -42,14 +42,13 @@ func (r *Resource) Create(
 	}
 
 	// Generate API request body from plan
-	createRequest := &packngo.SSHKeyCreateRequest{
-		Label:     plan.Name.ValueString(),
-		Key:       plan.PublicKey.ValueString(),
-		ProjectID: plan.ProjectID.ValueString(),
+	createRequest := &metalv1.SSHKeyCreateInput{
+		Label: plan.Name.ValueStringPointer(),
+		Key:   plan.PublicKey.ValueStringPointer(),
```

Comment:
I forget how the API & SDK are structured around this, but if the project ID is not an input parameter for creating a project SSH key, then we need to be passing in the project ID somewhere else (possibly using a different operation than `SSHKeysApi.CreateSSHKey`).

Reply:
Looked into the SDK, and for a project SSH key we need to use [`CreateProjectSSHKey`](https://github.com/equinix/equinix-sdk-go/blob/main/services/metalv1/api_ssh_keys.go#L59)